src/components/Layout/Header/NavBar/KeepKey/ChangePassphrase.tsx

```diff
@@ -72,7 +72,7 @@ export const ChangePassphrase = () => {
     // Trigger a refresh of the wallet metadata only once the settings have been applied
     // and the previous wallet meta is gone from the store
     dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
-    connect(KeyManager.KeepKey)
+    connect(KeyManager.KeepKey, false)
```

Comment:
q: do we really need this? I feel like the fact that it's an mipd should be managed under the hood, it doesn't seems abstracted (I might be wrong but let's double check)

Reply:
As discussed, would prefer to keep the optional arg for the time being - ensures we consistently pass it when we need to, which we could absolutely forget for e.g Coinbase

Reply:
Same thing here, piggybacking on that, we don't know what we don't know and risks-wise it just makes more sense to keep this declarative and consumer-driven as an initial PR risks-wise, though could absolutely remove in a follow-up and centralize mipd lookups.
Only thing we'll have to be cognicent about is to ensure the ones we want as first-class (Coinbas and Keplr) are still using first-class hdwallet, despite a mipd existing for these.

Reply:
Alright